tests/braille-specs/it-it-comp6.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright © 2025 by Tommaso Nonis <tommaso@visiondept.it>
```

Comment:
I happen to have made a test file for Italian too, for https://github.com/liblouis/liblouis/issues/1783 (but not committed yet). It tests both it-it-comp6.utb and it-it-comp8.utb, and I called it "it.yaml".

Reply:
See https://github.com/liblouis/liblouis/pull/1795.